tasks/investigate_boost_spike.py

```
@@ -0,0 +1,220 @@
+        return None
+
+    try:
+        with open(data_file, "r", encoding="utf-8") as f:
+            content = f.read()
+        return content
+    except Exception as e:
+        console.print(f"[yellow]Warning: Could not read {data_file}: {e}[/yellow]")
+        return None
+
+
+def create_research_query(distribution_data: str = None) -> str:
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)